### PR TITLE
[subject page] add block level per capita option

### PR DESCRIPTION
**Pull request by chejennifer**
- UI changes to allow for block level per capita option

screen-recording-2023-07-27T124816 969

### CODE REVIEW

File: static/js/components/subject_page/stat_var_provider.ts [hunk 1]
```diff
@@ -27,11 +28,23 @@ export class StatVarProvider {
...
+    return {
+      ...this._statVarSpecMap[key],
+      denom: blockDenom || this._statVarSpecMap[key].denom,
+    };
   }
 
-  getSpecList(keys: string[]): StatVarSpec[] {
-    return keys.map((k) => this._statVarSpecMap[k]);
+  // Gets the stat var spec for a list of keys. If blockDenom is non empty, set
+  // the denom each stat var spec in the list.
```
Comment (juliawu) [inline static/js/components/subject_page/stat_var_provider.ts:44]: nit: "the denom **of** each stat var spec"

**Review by juliawu — APPROVED** [review]
Awesome! Thanks for the additional tile cleanup.

File: static/js/components/tiles/bar_tile.tsx [hunk 2]
```diff
@@ -257,6 +258,7 @@ function rawToChart(
         const svUnit = getUnit(raw.facets[stat.facet]);
         unit = unit || svUnit;
       }
+      console.log(props.statVarSpec);
```
Comment (pradh) [inline static/js/components/tiles/bar_tile.tsx:261]: drop?

**Review by pradh — APPROVED** [review]
Very nice!

File: static/js/components/tiles/bar_tile.tsx (hunk 2 above)
Comment (chejennifer) [reply static/js/components/tiles/bar_tile.tsx]: good catch!

**Pull request merged**